<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Haberleşme Eğitim Kiti - HaKi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>MS Reference Sans Serif</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>KAYDET</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/record.png</normaloff>icons/record.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>MS Reference Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>Choose Center Frequency in MHz:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>MS Reference Sans Serif</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>RECORDING OPTIONS</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="3">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>Bandwidth:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QRadioButton" name="radioButton_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>1.024 MHz</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>Duration:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="radioButton_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>15 s</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QRadioButton" name="radioButton_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>60 s</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>2.4 MHz</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QRadioButton" name="radioButton_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>30 s</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QSlider" name="horizontalSlider">
              <property name="minimum">
               <number>24</number>
              </property>
              <property name="maximum">
               <number>1700</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>24 MHz</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="radioButton_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>2.048 MHz</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>MS Reference Sans Serif</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string> Recording Progress:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QProgressBar" name="progressBar">
            <property name="font">
             <font>
              <family>MS Reference Sans Serif</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="1">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Haberleşme Kiti</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icons/HaKi_white.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Ana Ekran</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButton_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>RECORD</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/binary-code.png</normaloff>icons/binary-code.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>200</width>
            <height>150</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="toolButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ISM</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/116053.png</normaloff>icons/116053.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>WBFM</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/1302.png</normaloff>icons/1302.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="toolButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>NBFM</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/34_walkie_talkie.png</normaloff>icons/34_walkie_talkie.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="toolButton_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ADS-B</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/2322549.png</normaloff>icons/2322549.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>200</width>
            <height>150</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>MS Reference Sans Serif</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Haberleşme Kiti</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icons/HaKi_white.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>50</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
